components/add_groups_to_channel_modal/add_groups_to_channel_modal.test.jsx

```diff
@@ -86,9 +89,9 @@ describe('components/AddGroupsToChannelModal', () => {
         setTimeout(() => {
             expect(instance.handleResponse).toBeCalledTimes(2);
         }, 0);
```

Comment:
I think this part can be refactored. Not sure why it's written the way it is but when I tried locally, seemed to pass fine.
```suggestion
          expect(instance.handleResponse).toBeCalledTimes(2);
```